### PR TITLE
Kaf 120 write out read me

### PR DESCRIPTION
**Pull request by jeb-stone**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,128 @@
...
+
+All other brokers will be listed in your feed and can be expanded to view metrics with the click or press of a button. 
+
+---
+
+## Install and Run
+To begin using kafkAlerts, navigate to <a href="http://www.kafkAlerts.com">kafkalerts.com</a> and create an account. 
+- Make sure that Prometheus is running and connected to your cluster. 
+- Click on the user menu icon in the top right.
+- Click on 'Connect Cluster'
```
Comment (mezzocarattere) [inline README.md:39]: missing period after the final ' (for consistency)

File: src/pages/landingPage/components/Hero.jsx
Comment (ian-flynn) [inline src/pages/landingPage/components/Hero.jsx:1]: change this to old version, span, h1 and p tag. Instead of h1 h2 h3

**Review by ian-flynn — CHANGES_REQUESTED** [review]

**Review by mezzocarattere — APPROVED** [review]
lgtm

**Review by ian-flynn — APPROVED** [review]
amazing, you've done it again

**Pull request merged**